src/main.rs

```diff
@@ -9,78 +9,82 @@ use std::str::FromStr;
+			Arg::with_name("COLUMNS")
+				.long("columns")
+				.default_value("5")
+				.validator(check_arg_is_number)
+				.help("Number of columns of the generated map")
+				.display_order(1),
 		)
-		.arg(Arg::with_name("START_ROW")
-			.long("start_row")
-			.default_value("0")
-			.validator(check_arg_is_number)
-			.help("The row to start generating from")
-			.display_order(2),
+		.arg(
+			Arg::with_name("START_ROW")
+				.long("start_row")
+				.default_value("0")
+				.validator(check_arg_is_number)
+				.help("The row to start generating from")
+				.display_order(2),
 		)
-		.arg(Arg::with_name("START_COLUMN")
-			.long("start_column")
-			.default_value("0")
-			.validator(check_arg_is_number)
-			.help("The column to start generating from")
-			.display_order(3),
+		.arg(
+			Arg::with_name("START_COLUMN")
+				.long("start_column")
+				.default_value("0")
+				.validator(check_arg_is_number)
+				.help("The column to start generating from")
+				.display_order(3),
 		)
-		.arg(Arg::with_name("DELAY")
-			.long("delay")
-			.default_value("50")
-			.validator(check_arg_is_number)
-			.help("The ms delay between steps")
-			.display_order(4),
+		.arg(
+			Arg::with_name("DELAY")
+				.long("delay")
+				.default_value("50")
+				.validator(check_arg_is_number)
+				.help("The ms delay between steps")
+				.display_order(4),
 		)
-		.arg(Arg::with_name("DFS")
-			.long("dfs")
-			.help("Use the depth first search algorithm for maze generation [default]")
-			.display_order(5)
+		.arg(
+			Arg::with_name("DFS")
+				.long("dfs")
+				.help("Use the depth first search algorithm for maze generation [default]")
+				.display_order(5),
 		)
-		.arg(Arg::with_name("TREE")
-			.long("tree")
-			.help("Use the binary tree maze algorithm for maze generation")
-			.display_order(6)
+		.arg(
+			Arg::with_name("TREE")
+				.long("tree")
+				.help("Use the binary tree maze algorithm for maze generation")
+				.display_order(6),
 		)
-		.arg(Arg::with_name("PRIM")
-			.long("prim")
-			.help("Use Prim's algorithm for maze generation")
-			.display_order(7)
+		.arg(
+			Arg::with_name("PRIM")
+				.long("prim")
+				.help("Use Prim's algorithm for maze generation")
+				.display_order(7),
 		)
-		.arg(Arg::with_name("AB")
-			.long("ab")
-			.help("Use the Aldous-Broder algorithm for maze generation")
-			.display_order(8)
+		.arg(
+			Arg::with_name("AB")
+				.long("ab")
+				.help("Use the Aldous-Broder algorithm for maze generation")
+				.display_order(8),
 		)
-		.arg(Arg::with_name("DIV")
-			.long("div")
-			.help("Use the recursive division method for maze generation")
-			.display_order(9)
+		.arg(
+			Arg::with_name("DIV")
+				.long("div")
+				.help("Use the recursive division method for maze generation")
+				.display_order(9),
 		)
-		.group(ArgGroup::with_name("ALGORITHM").args(&[
-			"DFS",
-			"TREE",
-			"PRIM",
-			"AB",
-			"DIV",
-		]))
+		.group(ArgGroup::with_name("ALGORITHM").args(&["DFS", "TREE", "PRIM", "AB", "DIV"]))
```

Comment:
I kept these on separate lines to make it easier for git incase I wanted to revert one of the algorithms. I agree with Rustfmt that this can be on one line. But can we mark this as a special case?